src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -71,6 +71,13 @@ public CommandBase cancel() {
         return cmd;
     }
 
+    public CommandBase afterMatch() {
```

Comment:
`resetLift` is a more descriptive name. Can you also add a comment explaining what this actually does?